net-misc/ptpd/ptpd-2.3.0.ebuild

```diff
@@ -0,0 +1,84 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Header: $
+
+EAPI=5
+
+inherit autotools eutils flag-o-matic systemd
+
+DESCRIPTION="Precision Time Protocol daemon"
+HOMEPAGE="http://ptpd.sf.net"
+
+if [[ ${PV} == "9999" ]]; then
+	ESVN_REPO_URI="http://svn.code.sf.net/p/ptpd/code/trunk"
+	inherit subversion
```

Comment:
You should avoid calling `inherit` more than once. Usually people put the VCS eclass in a variable and do like `inherit autotools eutils ... ${VCS_ECLASS}`